### ISSUE TITLE
Can not install because of postinstall script.

### ISSUE THREAD

**Issue by jpickwell** [foray1010/ignore-sync#865]
It does not make sense for Husky to run when a user installs the `ignore-sync` package.

Node: 22.4.1
NPM: 10.9.2

NPM Output:
```
$ npm install -D ignore-sync
npm error code 127
npm error path /app/node_modules/ignore-sync
npm error command failed
npm error command sh -c husky install
npm error sh: line 1: husky: command not found
```

**Comment by foray1010** [comment]
We are using [pinst](https://github.com/typicode/pinst) to rename `postinstall` to `_postinstall` [before publish](https://github.com/foray1010/ignore-sync/blob/master/package.json#L33). There seems to be [some issues](https://github.com/npm/cli/issues/7322) on either `npm` or `pinst` side that it now does not work anymore.

Thanks for the fix!

**Comment by foray1010** [comment]
Should be fixed in `v8.0.0`

**Comment by djcsdy** [comment]
You shouldn't really run husky at all when a user installs the package. This change hides the error, but could easily break something or do something unexpected if the user does have husky installed, either globally or in a package that consumes ignore-sync.

It looks like you are publishing this package manually by running `yarn publish`, so ideally you should manually run `pinst --disable` before every `yarn publish`. Unfortunately there is no good way to automate this that I know of.

If you're publishing from CI or from a script then you can of course just add `pinst --disable` before the publish step in your script.
